docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: "3"
+version: "2"
```

Comment:
What's incompatible about v3?

Reply:
I had issues with running docker-compose v3. I dont think there is any incompatible with v3.

Reply:
I'll give it a try

Reply:
It works with v3 for me

Reply:
Just keep it v3. I think I have to update my docker-engine before I can use v3